packages/client/tests/functional/_utils/setupTestSuiteEnv.ts

```diff
@@ -68,12 +82,18 @@ async function copyPreprocessed(from: string, to: string, suiteConfig: Record<st
  *
  * @param conditionFromComment
  * @param suiteConfig
+ * @param clientMeta
  * @returns
  */
-function evaluateMagicComment(conditionFromComment: string, suiteConfig: Record<string, string>): boolean {
+function evaluateMagicComment(
+  conditionFromComment: string,
+  suiteConfig: Record<string, string>,
+  clientMeta: ClientMeta,
+): boolean {
   const script = new Script(conditionFromComment)
   const value = script.runInNewContext({
     ...suiteConfig,
+    clientMeta,
```

Comment:
Is it used anywhere?

Reply:
It isn't. I thought it would be useful for doing something like `// @ts-test-if: clientMeta.runtime !== 'edge'`, but now I see it won't — the types are always the same (and should be). I'll remove this.